packages/calcite-components/src/components/combobox/combobox.tsx

```diff
@@ -112,6 +119,15 @@ export class Combobox
    */
   @Prop({ reflect: true }) clearDisabled = false;
 
+  /**
+   * Controls the display behavior of multiple selected items.
+   * This property does not apply when selection-mode is set to "single" or "single-persist".
+   * - "show-all" displays every selected chip, wrapping to the next line if necessary
+   * - "single" displays a single chip indicating the total amount of selected items
+   * - "fit-to-line" only displays selected chips that fit horizontally, with a non-closable chip indicating additional selected items as needed
+   */
+  @Prop({ reflect: true }) displayMode: DisplayMode = "show-all";
```

Comment:
Could we use "all", "single", "auto" here? I think we could explain the "fit-to-line" in the value documentation string. I think the "show-" part of "show-all" is implied and "fit-to-line" is a little confusing.

Reply:
@ashetland @SkyeSeitz ☝️ thoughts?

Reply:
I love it. Nice simplification.

Reply:
I agree the "fit-to-line" value is tricky! Only confusion with "auto" is how we have used it in the system thus far. Feels like it might introduce a somewhat conflicting meaning here as keep to a single line when elsewhere I believe we consistently use it as a hug contents value speaking to width/height.

Would simply `display-mode="all | fit | single"` work?

Reply:
Based on a discussion, the team voted to use `selection-display: "all" | "fit" | "single"`, `all` being the default behavior to wrap all chips, `fit` to be the mode with the overflow chips on a single line, and `single` for just a single indicator chip.